docs/distributed-scheduling.md

```diff
@@ -0,0 +1,153 @@
+# Distributed Scheduling in TiCDC
+
+## Background
+
+TiCDC boasts high availability and horizontal scalability. To make this possible, TiCDC needs a distributed scheduling mechanism, a mechanism by which tables can be distributed across the nodes in the cluster and sustain failures of nodes. We still need a center of control, which we call the Owner, but the Owner should fail over very quickly should the old one fails.
```

Comment:
"but the Owner should fail over very quickly `should` the old one fails. "
Should we use `when` here?

Reply:
Please refer to https://www.oxfordlearnersdictionaries.com/definition/english/should?q=should meaning 14